lib/crystalball/source_diff/formatting_checker.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Crystalball
+  class SourceDiff
+    # Determinates if file_diff's patch contains changes for whitespaces or comments only
+    module FormattingChecker
+      class << self
+        # Returns `true` if file_diff's patch contains changes for whitespaces or comments only
+        #
+        # @param [Crystalball::SourceDiff::FileDiff] file_diff
+        # @return [TrueClass, FalseClass]
+        def pure_formatting?(file_diff)
+          return false unless STRIPABLE_FILES.include?(Pathname(file_diff.path).extname)
+
+          return false if file_diff.moved? || file_diff.deleted? || file_diff.new?
```

Comment:
I think those two early return checks can move to a separate method for clarity.